apps/website/astro.config.ts

```diff
@@ -27,24 +29,27 @@ export default defineConfig({
       ],
     }),
     react(),
+    astroExpressiveCode({
+      themes: [nightOwl],
+      frames: { showCopyToClipboardButton: false },
```

Comment:
The "copy to clipboard" button is shown on all code examples except the ThemeProvider page's examples. Should this also be made true?

Reply:
this comes from expressive-code and it looks different from the one in our LiveExample, so i've disabled it.

plus, since static examples are intended for shorter snippets and do not show the full code, there is no need to make the whole thing copyable

Reply:
> this comes from expressive-code and it looks different from the one in our LiveExample, so i've disabled it.

Ohh, I see, sounds good.

> plus, since static examples are intended for shorter snippets and do not show the full code, there is no need to make the whole thing copyable

At least for me, when the code snippets are small, I tend to copy the entire thing. That way I don't have to reconstruct the DOM for components that use the composition API (e.g. Tabs).

For full code, I only copy selective lines that are pertinent to my use case.

Reply:
then i'll flip it so that only static examples are copyable. i want to avoid the styling inconsistency that occurs when both are present

[`ffb3b0b`](https://github.com/iTwin/iTwinUI/pull/1748/commits/ffb3b0b4721b796957759f9e4b2024a68c787549)